scripts/cargo-rmc

```
@@ -56,6 +41,12 @@ def main():
     if not rmc.dependencies_in_path():
         return 1
 
+    if args.gen_symbols:
+        print("WARNING: --gen-symbols is not supported for cargo-rmc.")
+
+    if args.allow_cbmc_verification_failure:
+        print("WARNING: --allow-cbmc-verification-failure is not supported for cargo-rmc.")
```

nit: can we fail here instead of warn? Warnings are easy to overlook